src/sincere_singularities/utils.py

```diff
@@ -51,3 +52,17 @@ def load_json(filename: str, json_type: type[T]) -> T:
         typed_json: T = dacite.from_dict(json_type, loaded_json)
 
         return typed_json
+
+
+def check_similarity(first: str, second: str) -> float:
+    """
+    Measure of the strings' similarity as a float.
+
+    Args:
```

Comment:
when you annotate the function, do you still have to describe the parameters? I find it a bit redundant.

Reply:
Yes.

Reply:
OKay!

just that it's a bit of a rehearsal for me